src/algebra/order/floor.lean

```diff
@@ -531,6 +537,11 @@ lemma fract_lt_one (a : α) : fract a < 1 := sub_lt.1 $ sub_one_lt_floor _
 @[simp] lemma fract_one : fract (1 : α) = 0 :=
 by simp [fract]
 
+@[simp] lemma abs_fract : |int.fract a| = int.fract a := abs_eq_self.mpr $ fract_nonneg a
```

Comment:
Can't simp do this if you tag `int.fract_nonneg`?

Reply:
Of course; thanks!

I have kept the lemma but moved the `simp` tag to `int.fract_nonneg`.

Reply:
What about doing the same to `abs_one_sub_fract`?

Reply:
Unfortunately `simp` is not smart enough to deduce `abs_one_sub_fract` from `fract_lt_one`.